### ISSUE TITLE
AWSVerifiedPermissions service fails to compile

### ISSUE THREAD

**Issue by jbelkins** [awslabs/aws-sdk-swift#1047]
### Describe the bug

The `AWSVerifiedPermissions` service, which was introduced after AWS SDK release 0.18.0, includes a struct property named `set`.  `set` is also a reserved word in Swift.

The serialization extension for that struct tries to escape `` `set` `` in backticks, then combine it with the suffix `Container`, which results in an invalid Swift identifier and a compile error.

```
/Users/runner/work/aws-sdk-swift/aws-sdk-swift/Sources/Services/AWSVerifiedPermissions/models/Models.swift:136:18: error: found an unexpected second identifier in constant declaration; is there an accidental break?
        let `set`Container = try values.decodeIfPresent([VerifiedPermissionsClientTypes.AttributeValue?].self, forKey: .`set`)
                 ^
```

### Expected Behavior

Swift compiles correctly.

(More specifically, the generated Swift identifier should be `setContainer` with nothing backticked.)

### Current Behavior

Swift compiles with the error identified above.

### Reproduction Steps

Generate Swift code from latest models
Compile SDK including the `AWSVerifiedPermissions` service.

### Possible Solution

Fix generation of symbols from Swift reserved words.

### AWS SWIFT SDK version used

main / latest

### Compiler and Version used

Xcode 14.3.1

### Operating System and version

macOS 13.3

**Comment by jbelkins** [comment]
Release 0.19.0 will be performed excluding this service.  Once this work is complete, that service may be re-included in the next release.

**Comment by jbelkins** [comment]
Re-enabled & shipped in AWS SDK for Swift 0.20.0
